content/work/coachhub.rst

```diff
@@ -84,7 +79,7 @@ What They Needed
 ----------------
 
 ORCAS partnered with OddBird to build a custom,
-mobile-responsive web application —
+mobile-responsive web application –
```

Comment:
do we have to say mobile-responsive? How about just responsive? I blame Squarespace since they used that in their ads all the time :)